android/app/src/main/res/values/dimens.xml

```diff
@@ -174,4 +174,9 @@
   <dimen name="bookmark_purchase_img_width">280dp</dimen>
   <dimen name="square_block_size">168dp</dimen>
   <dimen name="about_logo">96dp</dimen>
+
+  <!--About Fragment -->
+  <dimen name="max_healine_width">260dp</dimen>
+  <dimen name="max_bullet_width">280dp</dimen>
```

Comment:
@Jean-BaptisteC @rtsisyk @arnaudvergnet is it possible to avoid hard-coding these widths?

Reply:
I have used constraint layout to improve layout.
maxWitdh is necessary to avoid bullet + text overlap OM icon
I have keep only one value for text.
Maybe I can remove hardcoded for size button